From 92e4cee73c2e9550f9421214190cf320d137c7a4 Mon Sep 17 00:00:00 2001
From: Nell <xianbin.zhu@spacemit.com>
Date: Sat, 12 Oct 2024 16:28:12 +0800
Subject: [PATCH 1009/1202] cpuidle: delete the dts node for cpuidle

The functionality of cpuidle is not yet complete, so we'd
better to delete the cpuidle node in dts to prevent this
feature from being turned on by other kernel configurations;
like CONFIG_SOC_VIRT

Change-Id: Id46f216dcb66ec40a0e9c13409715a8b897540a9
---
 arch/riscv/boot/dts/spacemit/k1-x.dtsi | 19 ++++++++++---------
 1 file changed, 10 insertions(+), 9 deletions(-)

diff --git a/arch/riscv/boot/dts/spacemit/k1-x.dtsi b/arch/riscv/boot/dts/spacemit/k1-x.dtsi
index 199c96874ef6..8983970bea75 100644
--- a/arch/riscv/boot/dts/spacemit/k1-x.dtsi
+++ b/arch/riscv/boot/dts/spacemit/k1-x.dtsi
@@ -64,7 +64,7 @@ cpu_0: cpu@0 {
 			d-cache-sets = <128>;
 			next-level-cache = <&clst0_l2_cache>;
 			mmu-type = "riscv,sv39";
-			cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
+			// cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
 			#cooling-cells = <2>;
 
 			cpu0_intc: interrupt-controller {
@@ -98,7 +98,7 @@ cpu_1: cpu@1 {
 			d-cache-sets = <128>;
 			next-level-cache = <&clst0_l2_cache>;
 			mmu-type = "riscv,sv39";
-			cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
+			// cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
 			#cooling-cells = <2>;
 
 			cpu1_intc: interrupt-controller {
@@ -132,7 +132,7 @@ cpu_2: cpu@2 {
 			d-cache-sets = <128>;
 			next-level-cache = <&clst0_l2_cache>;
 			mmu-type = "riscv,sv39";
-			cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
+			// cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
 			#cooling-cells = <2>;
 
 			cpu2_intc: interrupt-controller {
@@ -166,7 +166,7 @@ cpu_3: cpu@3 {
 			d-cache-sets = <128>;
 			next-level-cache = <&clst0_l2_cache>;
 			mmu-type = "riscv,sv39";
-			cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
+			// cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
 			#cooling-cells = <2>;
 
 			cpu3_intc: interrupt-controller {
@@ -200,7 +200,7 @@ cpu_4: cpu@4 {
 			d-cache-sets = <128>;
 			next-level-cache = <&clst1_l2_cache>;
 			mmu-type = "riscv,sv39";
-			cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
+			// cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
 			#cooling-cells = <2>;
 
 			cpu4_intc: interrupt-controller {
@@ -234,7 +234,7 @@ cpu_5: cpu@5 {
 			d-cache-sets = <128>;
 			next-level-cache = <&clst1_l2_cache>;
 			mmu-type = "riscv,sv39";
-			cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
+			// cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
 			#cooling-cells = <2>;
 
 			cpu5_intc: interrupt-controller {
@@ -268,7 +268,7 @@ cpu_6: cpu@6 {
 			d-cache-sets = <128>;
 			next-level-cache = <&clst1_l2_cache>;
 			mmu-type = "riscv,sv39";
-			cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
+			// cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
 			#cooling-cells = <2>;
 
 			cpu6_intc: interrupt-controller {
@@ -302,7 +302,7 @@ cpu_7: cpu@7 {
 			d-cache-sets = <128>;
 			next-level-cache = <&clst1_l2_cache>;
 			mmu-type = "riscv,sv39";
-			cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
+			// cpu-idle-states = <&CPU_NONRET> /*, <&CLUSTER_NONRET>, <&TOP_NONRET> */;
 			#cooling-cells = <2>;
 
 			cpu7_intc: interrupt-controller {
@@ -329,7 +329,7 @@ clst1_l2_cache: l2-cache1 {
 			cache-sets = <512>;
 			cache-unified;
 		};
-
+/*
 		idle-states {
 			CPU_NONRET: cpu-nonret {
 				compatible = "riscv,idle-state";
@@ -363,6 +363,7 @@ TOP_NONRET: top-nonret {
 				wakeup-latency-us = <1600>;
 			};
 		};
+*/
 	};
 
 	clocks {
-- 
2.47.0

